packages/calcite-components/src/components/combobox-item/combobox-item.scss

```
@@ -1,3 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-combobox-text-color: Specifies the text color of the component.
+ * @prop --calcite-combobox-text-color-hover: Specifies the text color of the component when hovered.
+ * @prop --calcite-combobox-item-background-color-active: Specifies the background color of the component when active.
+ * @prop --calcite-combobox-item-background-color-hover: Specifies the background color of the component when hovered.
+ * @prop --calcite-combobox-selected-icon-color: Specifies the color of the selected indicator icon.
+ * @prop --calcite-combobox-description-text-color: Specifies the text color of the component's description & short-heading.
```

WDYT about refactoring this a bit to highlight the props more?

```suggestion
 * @prop --calcite-combobox-description-text-color: Specifies the component's `description` and `shortHeading` text color.
```